src/views/Inventory.vue

```diff
@@ -33,6 +33,7 @@
               </ion-label>
             </ion-item>
           </ion-card>
+          <MoreJobs :jobs="moreJobs" :jobEnums="jobEnums" />
```

Comment:
Add v-if condition on the page to hide the `MoreJobs` section instead of adding it in the `MoreJobs` component

Reply:
Done sir